presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/main/grouphome/GroupHomeScreen.kt

```diff
@@ -72,6 +73,7 @@ fun GroupHomeScreen(
     onClickMyPage: () -> Unit,
     onClickGroupEnter: () -> Unit,
     onClickGroupMake: () -> Unit,
+    onClickPicFcm: () -> Unit,
```

Comment:
```suggestion
    onClickSendFcm: () -> Unit,
```
PicFcm 보다는 SendFcm 이 조금 더 직관적일 것 같은데 이런 이름은 어때?!

Reply:
아아아 오키오키 그게더 직관적이겠당 ㅎ_ㅎ 바로 수정할게~~~~